src/libmongoc/src/mongoc/mongoc-log-private.h

```diff
@@ -34,4 +34,14 @@ mongoc_log_trace_bytes (const char *domain, const uint8_t *_b, size_t _l);
 void
 mongoc_log_trace_iovec (const char *domain, const mongoc_iovec_t *_iov, size_t _iovcnt);
 
+BSON_BEGIN_DECLS
```

Comment:
Suggest removing `BSON_BEGIN_DECLS` and `BSON_END_DECLS`. Those are not needed for private definitions.

Aside: the macros conditionally define `extern "C" { ... }` to tell C++ compilers to treat the symbols as C declarations. But private headers are not expected to be used directly by C++ code.